lib/velveteen/worker.rb

```diff
@@ -64,15 +64,10 @@ def generate_schema(routing_key)
       File.expand_path(schema_file_path, Config.schema_directory)
     end
 
-    def message_schema
-      if self.class.message_schema
-        File.expand_path(self.class.message_schema, Config.schema_directory)
-      end
-    end
-
     def maybe_validate_message!
-      if message_schema
-        errors = JSON::Validator.fully_validate(message_schema, message.data)
+      if routing_key
```

Comment:
I think we can make the assumption that a routing key exists, because otherwise the worker wouldn't have gotten a message to work and therefore couldn't be publishing anything. There's probably a number of specs that would need to be updated to account for that expectation.

Reply:
Dope, I was wondering if there was cases without a routing key due to the specs, but it makes sense that there isn't one. Updating.